README.md

```diff
@@ -1 +1,112 @@
-# Room Access rules
\ No newline at end of file
+# Room Access rules
+
+This module implements handling around the `im.vector.room.access_rules` state event, which spec is described below.
+
+## `im.vector.room.access_rules`
+
+Restricts the access to a room based on the selected preset. Body:
+
+```json
+{
+    "rule": "<rule>"
+}
+```
+
+`<rule>` is either `restricted`, `unrestricted` or `direct`.
+
+The implementation of the different presets lives in the
+`synapse.third_party_rules.access_rules` module.
+
+### `restricted` preset
+
+Default preset for non-direct rooms.
```

Comment:
I was a bit confused as to whether `direct` here referred to the "direct" preset, or 1:1 rooms. But looks like it's the latter.

```suggestion
Default preset for non-direct rooms (i.e. rooms created with `"is_direct": false`).
```

Reply:
Edited your suggestion since `is_direct` is an optional parameter defaulting to `false` iirc